service/src/main/java/com/ge/predix/acs/attribute/readers/ExternalAttributeReader.java

```diff
@@ -120,7 +120,7 @@ CachedAttributes getAttributesFromAdapters(final String identifier) throws Attri
             try {
                 attributesResponse = this.getAdapterOauth2RestTemplate(adapterConnection).
                         getForEntity(adapterUrl, AttributesResponse.class).getBody();
-            } catch (final Throwable e) {
+            } catch (final Exception e) {
```

Comment:
From SonarQube Static Analyzer:

Throwable is the superclass of all errors and exceptions in Java.

Error is the superclass of all errors, which are not meant to be caught by applications.

Catching either Throwable or Error will also catch OutOfMemoryError and InternalError, from which an application should not attempt to recover.